SonarQube.Client/Requests/IRequestFactorySelector.cs

```diff
@@ -42,7 +42,7 @@ public IRequestFactory Select(bool isSonarCloud, ILogger logger)
             }
 
             logger.Debug("Selected SonarQube request factory");
-            return DefaultConfiguration.Configure(new RequestFactory(logger));
+            return DefaultConfiguration.ConfigureSonarQube(new RequestFactory(logger));
```

Comment:
Note: this class doesn't yet return a SonarCloud-specific factory.
That change will break some of the existing tests, so I left it for another PR.